crds/AzureApplicationGatewayLoadDistributionPolicy.yaml

```diff
@@ -0,0 +1,57 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: azureapplicationgatewayloaddistributionpolicies.appgw.ingress.azure.io
+spec:
+  group: appgw.ingress.azure.io
+  scope: Namespaced
+  names:
+    kind: AzureApplicationGatewayLoadDistributionPolicy
+    plural: azureapplicationgatewayloaddistributionpolicies
+    singular: azureapplicationgatewayloaddistributionpolicy
+    shortNames:
+      - agldp
+  versions:
+    - name: v1
+      served: true
+      storage: true
+      schema:
+        openAPIV3Schema:
+          type: object
+          properties:
+            spec:
+              type: object
+              properties:
+                targets:
+                  description: "An array of services that contains information for how the service should be used for load balancing and active passive support in Application Gateway"
+                  type: array
+                  items:
+                    type: object
+                    properties:
+                      backend:
+                        type: object
+                        properties:
+                          service:
+                            description: "service object model"
+                            type: object
+                            properties:
+                              name:
+                                type: string
+                                description: "same of service"
+                              port:
+                                type: object
```

Comment:
description for Port: `Port of the referenced service. A port name or port number`

Reply:
you can reference service related description from the v1.Ingress API definition

https://github.com/kubernetes/api/blob/master/networking/v1/types.go#L438